draft-ietf-jsonpath-base.md

```diff
@@ -1502,10 +1502,21 @@ The well-typedness of function expressions can now be defined in terms of this t
 
 For a function expression to be well typed:
 
-1. its declared type must be well typed in the context in which it occurs, and
-2. its arguments must be well typed for the declared type of the corresponding parameters.
+1. its function name must be registered (see {{iana-fnex}}),
+2. its declared type must be well typed in the context in which it occurs, and
+3. its arguments must be well typed for the declared type of the corresponding parameters.
 
-(1) As per the grammar, a function expression can occur in three different
+(1) The registered set of function extensions is the source of knowledge regarding the return and
+parameter types of a function extension.
+
+Although a function expression being treated as well typed implies its function name is registered,
+the reverse is not necessarily true. Implementations must support the function extensions defined
```

Comment:
I don't know what the reverse is: registered ➔ well-typed?  Of course not.
Or is it the inverse?
Better be explicit?

Reply:
Fixed in e25a509952eeae5c66facd5fd98a073c844b1b20.